.gitignore

```diff
@@ -61,4 +61,5 @@ requirements/private.in
 requirements/private.txt
 
 # IDE related
-.idea/*
+.idea/
+.vscode/
```

Comment:
Can you move this to a separate PR or commit?

Reply:
Removed from this PR.